test/index.html

```diff
@@ -155,6 +160,46 @@ <h1>Some Sample CFPB Charts</h1>
                      data-chart-type="bar">
                      This is the chart description.
                 </div>
+                <hr>
+                <div class="cfpb-chart"
+                     data-chart-color="navy"
+                     data-chart-description="Indexed number of people with inquiries."
+                     data-chart-source="consumer-credit-trends/auto-loans/inquiry_rate_AUT.csv"
+                     data-chart-title="Inquiry rate auto"
+                     data-chart-type="line">
+                     Indexed number of people with inquiries.
```

Comment:
Is this the displayed chart title? If so, no period at the end of it. (See https://cfpb.github.io/design-manual/data-visualization/chart-components.html#titles for more info.) Same change for the other inquiry charts that follow.

Reply:
This is what shows when JavaScript is disabled for the charts. In production it's the same as the title. Thinking about it, it should probably be something like "Display of this chart requires JavaScript be enabled."

![screen shot 2018-08-27 at 4 57 59 pm](https://user-images.githubusercontent.com/704760/44685839-74c8fa00-aa1a-11e8-97d5-8374ec10ec96.png)